src/components/ExploreMovieCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import { Movie, MoviePreference, PreferenceType, User } from "../types";
+
+import { Card, Image } from "semantic-ui-react";
+
+import ExploreMovieCardBottom from "./ExploreMovieCardBottom";
+
+const ExploreMovieCard = (props: {
+  movie: Movie;
+  changeMoviePreference: (
+    externalMovieId: string,
+    preferenceType: PreferenceType
+  ) => void;
+  moviePreference: MoviePreference | null;
+}): JSX.Element => {
+  const movie = props.movie;
```

Comment:
You can do ```
const ExploreMovieCard = ({movie, changeMoviePreference, moviePreference}: {
  movie: Movie,
  changeMoviePreference: (externalMovieId: string, preferenceType: PreferenceType) => void,
  moviePreference: MoviePreference | null
}): JSX.Element =>```

if you want, which will avoid needing to set all those variables. Although b/c of typescript, it ends up being about as verbose...

Reply:
Oh wow, github code blocks suck

Reply:
Done